src/components/swimlanes/swimLanes.tsx

```diff
@@ -0,0 +1,110 @@
+  Link,
+} from "@nypl/design-system-react-components";
+import styles from "./Swimlanes.module.css";
+import React from "react";
+import { imageURL } from "@/utils/utils";
+import appConfig from "appConfig";
+
+const SwimLanes = ({ lanesWithNumItems }) => {
+  return (
+    <>
+      {lanesWithNumItems.map((lane, key) => (
+        <Box data-testid={lane.slug} key={lane.rank} mb="xxl">
+          <Flex alignItems="baseline">
+            <Heading id={`row-heading-${lane.slug}`} level="h2" size="heading3">
+              {lane.title}
+            </Heading>
+            <Spacer />
+            <Link
+              id={`row-see-more-${lane.slug}`}
+              type="standalone"
+              href={`${appConfig.DC_URL}collections/lane/${lane.slug}`}
+              aria-label={`See more ${lane.title.toLowerCase()}`}
+              sx={{
+                display: { sm: "none", md: "inline" },
+                fontWeight: "500",
+                alignItems: "center",
+              }}
+            >
+              See more
+            </Link>
+          </Flex>
+          {lane.collections && lane.collections.length > 0 && (
+            <SimpleGrid columns={4} id={`grid-${key}`}>
+              {lane.collections.map((collection, index) => (
+                <Card
+                  key={index}
+                  id={`card-${key}-${index}`}
+                  imageProps={{
+                    alt: "",
+                    aspectRatio: "twoByOne",
+                    src: imageURL(collection.image_id, "full", "288,", "0"),
+                  }}
+                >
+                  <CardHeading
+                    id={`row-card-heading-${key}-${index}`}
+                    level="h3"
+                    size="heading5"
+                    className={styles.collectiontitle}
+                    url={collection.url}
+                    style={{
+                      display: "-webkit-box",
+                      WebkitLineClamp: 3,
+                      lineClamp: 3,
+                      WebkitBoxOrient: "vertical",
+                      boxOrient: "vertical",
+                      overflow: "clip",
+                    }}
+                  >
+                    {collection.title}
+                  </CardHeading>
+                  <CardContent>
+                    <Text
+                      size="subtitle2"
+                      sx={{
+                        fontWeight: "400",
+                        display: { sm: "none", md: "inline" },
+                      }}
+                    >
+                      {collection.numItems} items
+                    </Text>
+                  </CardContent>
+                </Card>
+              ))}
+            </SimpleGrid>
+          )}
+          <Link
+            id={`row-see-more-${lane.slug}-mobile`}
+            type="standalone"
+            href={`${appConfig.DC_URL}collections/lane/${lane.slug}`}
+            aria-label={`See more ${lane.title.toLowerCase()} mobile`}
```

Comment:
We chatted about this but here are more details: you can use `aria-hidden` so that a screenreader ignores that flagged element. Adding that will help remove the "mobile" from the aria-label here.

This _should_ be conditional, however, because in the mobile context you do want this link to be picked up by screenreaders and the desktop link to be ignored.

The DS has a helper hook `useNYPLBreakpoints` https://nypl.github.io/nypl-design-system/reservoir/v2/?path=/docs/hooks-usenyplbreakpoints--docs that can help with that. It uses the `matchMedia` API to check the current width of the viewport. This will tell you if it "isLargerThanMobile" and you can then apply the `aria-hidden={true}` appropriately.

Another approach, which is an example we give in the doc, is to only render this `Link` on mobile and when `isLargerThanMobile` is false, then render `null` when it's true. The reverse would be the case for the other link above. Both approaches are similar and I don't think any is better than another.

Reply:
Change made and then changed back– the Axe issue was with the ID, which can't be a duplicate anyways. `display: none` implies `aria-hidden: true` to the screenreader, which Clare just explained on the ticket. The more you know!

Reply:
Great. I was confused because of the failing test you mentioned and having the same labels. I didn't know it was a duplicate id issue so this update makes sense.